react-common/components/profile/Profile.tsx

```diff
@@ -44,5 +46,24 @@ export const Profile = (props: ProfileProps) => {
             userState={userBadges}
             onBadgeClick={onBadgeClick}
         />}
+
+        {profileSmall &&
+            <div className="profile-info-container">
+                <p className="profile-info">
+                    {lf("Now that you're logged in, your projects will be automatically saved to the cloud so you can access them from any device! ")}
+                    {lf("Learn more at ")}
+                    <a href="https://arcade.makecode.com/identity/cloud-sync" target="_blank" rel="noopener noreferrer" tabIndex={0}>{lf("Cloud Sync ")}</a>
```

Comment:
We shouldn't hardcode target in the PXT.

Reply:
As in this link shouldn't be the full arcade.makecode.com?

Reply:
Yes, there is a way to get the full URL.

Reply:
@jwunderl ?

Reply:
Ah I was writing out a not for it down below, we ought to add the ref pages to /common-docs so they're available per target (e.g. see common-docs/faq and how it is available or overridden in any target